code/modules/jobs/job_types/corporate_affairs_agent.dm

```diff
@@ -0,0 +1,63 @@
+/datum/job/caa
+	title = "Corporate Affairs Agent"
+	flag = CAA
+	department_head = list("Captain")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "Central Command"
+	selection_color = "#ddddff"
+	chat_color = "#50C878"
+	exp_requirements = 840
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/caa
+
+	access = list(ACCESS_CAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO,
+				ACCESS_HEADS)
+	minimal_access = list(ACCESS_CAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO,
+				ACCESS_HEADS)
+	paycheck = PAYCHECK_MEDIUM
```

Comment:
ACCESS_HEADS is for red alert control panel access mainly, they should be getting ACCESS_BRIDGE if you want them to have bridge access or none otherwise.

Reply:
There is no ACCESS_BRIDGE. ACCESS_HEADS is the access that allows entering the bridge.

Reply:
Splits off bridge access from ACCESS_HEADS into its own thing (ACCESS_BRIDGE) and added it to the CAA ID instead of ACCESS_HEADS